servers.json

```diff
@@ -788,5 +788,12 @@
         "addresses": [
             "originrealms.com"
         ]
+    },
+    {
+        "id": "ventry",
+        "name": "Ventry Combo",
+        "addresses": [
+            "play.ventry.hu"
```

Comment:
Please remove the `play.` prefix from the domain;

The system automatically detects sub-domains, [click here](https://github.com/LunarClient/ServerMappings/blob/master/README.md#ip-addresses) for more information on ip-addresses.

Reply:
How to change?